rootfs/api/models.py

```diff
@@ -168,7 +168,7 @@ class App(UuidAuditedModel):
     """
 
     owner = models.ForeignKey(settings.AUTH_USER_MODEL)
-    id = models.SlugField(max_length=64, unique=True, default=select_app_name,
+    id = models.SlugField(max_length=24, unique=True, default=select_app_name,
```

Comment:
This probably needs to add a Django South migration for the schema change. (I tried to test it but was able to create app names much longer than 24 chars since the database schema was defined earlier with 64 chars.)